AEPCore/Sources/eventhub/ExtensionRuntime.swift

```diff
@@ -48,8 +48,9 @@ public protocol ExtensionRuntime {
     /// 2. If this extension has previously published a shared state, shared state will be versioned at the latest
     /// - Parameters:
     ///   - data: Data for the `SharedState`
+    ///   - xdmData: XDM data for the `SharedState`
     ///   - event: `Event` for which the `SharedState` should be versioned
-    func createSharedState(data: [String: Any], event: Event?)
+    func createSharedState(data: [String: Any], xdmData: [String: Any]?, event: Event?)
```

Comment:
Same comment here, if we want both data and xdmData to be provided they shouldn't be optional.